test/conftest.py

```diff
@@ -41,3 +47,7 @@ async def page(context):
     page = await context.newPage()
     yield page
     await page.close()
+
+@pytest.fixture(autouse=True)
+def start_http_server():
+    _thread.start_new_thread(start_server, ())
```

Comment:
To me, it seems like it would be better to use the higher-level [threading](https://docs.python.org/3/library/threading.html#module-threading) module instead of this one.

> The threading module provides an easier to use and higher-level threading API built on top of this module.

Since it starts with an underscore it seems like it's not meant to be so commonly used.

I think you should be able to write something like:
```py
threading.Thread(target=start_server).start()
```